src/modules/real-time-metrics/filters/load-info-available-filters.js

```diff
@@ -44,14 +43,15 @@ const datasetListQuery = `query {
  * @return {Object} The processed available filters information.
  */
 export default async function LoadInfoAvailableFilters() {
-  if (cancelRequest) cancelRequest.cancel()
-  const ReportsRequestToken = Axios.CancelToken
-  cancelRequest = ReportsRequestToken.source()
+  if (abortController) abortController.abort()
+  abortController = new AbortController()
 
   const graphqlQuery = { query: datasetListQuery }
 
-  const beholderService = new BeholderService({ cancelRequest })
-  const { __type } = await beholderService.gql(graphqlQuery)
+  const { __type } = await loadRealTimeMetricsData({
```

Comment:
rename __type to a more described name and avoid missing project naming format for variables